javascripts/buzz.js

```diff
@@ -224,6 +224,10 @@ app.buzz = {
 
           app.buzz.infiniteScrollCalled = true;
           var from = $('.buzz-item').length + 1;
```

Comment:
I think the real fix is to change this to:

    var from = $('.buzz-item').length